packages/cloud-cognitive/src/components/Card/Card.js

```diff
@@ -90,9 +90,17 @@ export let Card = forwardRef(
                 size={actionsPlacement === 'top' ? 'sm' : 'field'}
                 iconDescription={iconDescription}
                 kind="ghost"
+                href={href}
               />
             );
           }
+          if (href) {
+            return (
+              <a key={id} className={`${blockClass}__icon`} href={href}>
```

Comment:
Should we include an `onClick` here if they happen to provide both? Like we do for the button case above.

```suggestion
              <a
                key={id}
                className={`${blockClass}__icon`}
                onClick={onClick}
                href={href}
              >
```

Reply:
yeah that makes sense to me. good call 👍